addons/api/addon/models/target.js

```diff
@@ -318,6 +318,34 @@ export default class TargetModel extends GeneratedTargetModel {
       options
     );
   }
+  /**
+   * save target with filters based on egress/ingress toggle
+   * @param {object} target
+   * @param {object} egress_worker_filter
+   * @return {Promise}
+   */
+  async saveWithToggles(target, egressEnabled = true) {
```

Comment:
After reviewing, this should live in the route.  "egressEnabled" is a mostly UI concept and doesn't make sense in the API layer.